docs/team/gnoossk.md

```diff
@@ -0,0 +1,29 @@
+---
+  layout: default.md
+  title: "John Doe's Project Portfolio Page"
```

Comment:
I think this should be Hugh's Project Portfolio.

Reply:
You're correct Ansel. Thank you for pointing it out. 😺